src/nodes/chunk_dispatch/chunk_dispatch.c

```diff
@@ -106,8 +108,27 @@ ts_chunk_dispatch_get_chunk_insert_state(ChunkDispatch *dispatch, Point *point,
 			chunk = ts_hypertable_create_chunk_for_point(dispatch->hypertable, point, &found);
 		}
 		else
+		{
 			found = true;
 
+			if (ts_chunk_is_compressed(chunk) && chunk->data_nodes == NIL && slot)
```

Comment:
slot can't be null, right? Might be more clear to convert the above Assert into Ensure if it's just to prevent segfaults

Reply:
there is an assert earlier that is never hit and i think that is leftover from an earlier version